README.md

```diff
@@ -38,3 +38,18 @@ highlighting, anonymity, and secure authentication.
 - Preferably use `ROCKET_ENV=prod` and set a secret key in `Rocket.toml`
   - See [Rocket docs](https://rocket.rs/guide/configuration/)
 - Reverse proxy and handle `/static/` with a webserver and not the included route
+
+### Usage (docker)
+
+- Clone the repo (`--recursive` for submodules)
+- `echo -e "DATABASE_URL=postgres://paste:paste@db/paste\n\nROCKET_ADDRESS=0.0.0.0" > .env`
```

Comment:
possibility to let ROCKET_ADDRESS only target internal network addresses? default is `localhost`, which means nginx can't communicate with it. If this is a bind that'll be problematic, maybe `nginx` will work .. dunno.

Reply:
Why are we setting Rocket's address in .env? There's options in [`Rocket.toml`](https://rocket.rs/guide/configuration/#rockettoml).

Reply:
Rocket.toml is versioned by git.

Reply:
As discussed, this should be in a separate folder with other config files (see `master` branch README).